k8s/dev-on-kind.md

```diff
@@ -0,0 +1,104 @@
+# Deploying zqd on Kind
+
+This assumes you have already followed the instructions in [Setting up a Kind cluster for zqd](kind-setup.md).
+
+## Build and upload docker image
+To build and push the zqd image to the local Docker repo that is deployed on Kind, use:
+```
+make docker-push-local
+```
+
+## Install with Helm
+Helm is used to deploy the zqd image. Use:
+```
+make helm-install
+```
+To run Helm with the correct command line flags.
+
+After helm-install, you can check the status of your install with:
+```
+helm ls
+```
+If you want to redeploy in you test env, first uninstall the zqd instance with:
+```
+helm uninstall zqd
+```
+To check the status of your running pod in your namespace, use:
+```
+kubectl get pod
+```
+To see the unique name of your running zqd pod. Copy that name for the following troubleshooting steps. If the status of the pod in 'Error' of 'ImagePullBackoff' (or something else not good), then you can get details with:
+```
+kubectl describe pod zqd-56b46985fc-bqv87
+kubectl logs zqd-56b46985fc-bqv87 -p
+```
+Edit the commands to use your pod name.
+
+## Creating test data in S3 using zar
+Before using zapi to access the running zqd, we use `zar import` in another console to copy sample data from our zq repo into s3. You must have an AWS account and the AWS CLI on your desktop machine to do this. Change the directory name to match your s3 bucket.
+```
+zar import -R s3://brim-scratch/mark/sample-http-zng zq-sample-data/zng/http.zng.gz
```

Comment:
You no longer have to do this `zar import` rigamorale. Data can be imported into a thusly:
```
zapi new -k archivestore http-space
zapi -s http-space post zq-sample-data/zng/http.zng.gz
```

Reply:
Eventually, once archivestore becomes the default, you'll be able to create a post new data with one command.

Reply:
Great, I am removing this subheading, since it is no longer needed.